### PR TITLE
Document new set_state_service config for light platforms that have it

### PR DESCRIPTION
**Pull request by gjbadros**
Documentation for https://github.com/home-assistant/home-assistant/pull/29380 (PR 39380)

**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/flux.markdown [hunk 1]
```diff
@@ -27,6 +27,12 @@ switch:
     lights:
       - light.desk
       - light.lamp
+  - platform: flux
```
Comment (klaasnicolaas) [inline source/_integrations/flux.markdown]: Please remove the added example

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Please rebase this PR to the `next` branch

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Rebased PR onto `next`. (@gjbadros Changing the target branch is not sufficient, you had to git rebase, which I have done for you now 😉 )

**Pull request closed without merging**